src/display/annotation_helper.js

```diff
@@ -269,6 +269,45 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
     return container;
   }
 
+  function getHtmlElementForVideoAnnotation(item) {
+    var container = initContainer(item);
+    container.className = 'annotVideo';
+
+    container.style.borderColor = item.colorCssRgb;
```

Comment:
The border styles are already handled, so I don't see why borders should be adjusted here.